docs/zh/openmldb_sql/ddl/CREATE_INDEX_STATEMENT.md

```diff
@@ -40,15 +39,14 @@ OptionEntry ::=
 
 ```
 
-
-
 ## **示例**
 ```SQL
 CREATE INDEX index2 ON t5 (col2);
 -- SUCCEED
 ```
 ```{note}
-如果不指定Options, 创建的索引就没有指定`TS`列，因此不能用在需要上线的SQL中。
+1. 如果不指定Options, 创建的索引就没有指定`TS`列，因此不能用在需要上线的SQL中。
+2. 指定`TS`列的类型只能是BitInt或者Timestamp
```

Comment:
bitint? And smallint, int is ok to be ts? ref https://github.com/4paradigm/OpenMLDB/blob/3d790513489a468790f2ba00508cf289c1f3ec75/src/codec/codec.cc#L743
Is it disabled in user level?